src/components/Board/index.spec.js

```diff
@@ -1492,6 +1493,387 @@ describe('<Board />', () => {
           })
         })
       })
+
+      describe('when the component does not receive a custom header column template', () => {
+        const onCardNew = jest.fn()
+        const onNewCardConfirm = jest.fn(column => new Promise(resolve => resolve({ id: 999, ...column })))
+
+        describe('when the component does not receive "allowAddCard" prop', () => {
+          beforeEach(() => {
+            mount({ allowAddCard: false, onNewCardConfirm, onCardNew })
+          })
+
+          it('does not render the card adder', () => {
+            expect(subject.queryByText('+')).not.toBeInTheDocument()
+          })
+        })
+
+        describe('when the component does not receive the "onNewCardConfirm" prop', () => {
+          beforeEach(() => {
+            mount({ allowAddCard: true, onCardNew: () => {} })
+          })
+
+          it('does not render the column adder', () => {
+            expect(subject.queryByText('+')).not.toBeInTheDocument()
+          })
+        })
+
+        describe('when the component receives both the "allowAddCard" and "onNewCardConfirm" props', () => {
+          describe('when the user adds a new card', () => {
+            beforeEach(async () => {
+              mount({ allowAddCard: true, onNewCardConfirm, onCardNew })
+
+              fireEvent.click(subject.queryAllByText('+')[0])
+              fireEvent.change(subject.container.querySelector('input[name="title"]'), {
+                target: { value: 'Card title' }
+              })
+              fireEvent.change(subject.container.querySelector('input[name="description"]'), {
+                target: { value: 'Card description' }
+              })
+              fireEvent.click(subject.queryByText('Add'))
+              await waitForElement(() => subject.container.querySelector('[data-testid="card"]:nth-child(3)'))
+            })
+
+            it('calls the "onNewCardConfirm" passing the new card', () => {
+              expect(onNewCardConfirm).toHaveBeenCalledTimes(1)
+              expect(onNewCardConfirm).toHaveBeenCalledWith({
+                title: 'Card title',
+                description: 'Card description'
+              })
+            })
+
+            it('renders the new card using the id returned on "onNewCardConfirm"', () => {
+              expect(subject.queryAllByTestId('card')).toHaveLength(4)
+            })
+
+            it('renders the card placeholder', () => {
+              expect(subject.queryAllByText('+')).toHaveLength(2)
+            })
+
+            it('adds a new card on the bottom of the column', () => {
```

Comment:
Since we have a context to talk about the position, I think we should avoid talking about the position here.
```suggestion
            it('adds a new card on column', () => {
```